maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractMapVersionsPhase.java

```diff
@@ -347,16 +362,42 @@ private String getContextString( ReleaseDescriptor releaseDescriptor )
         return "new development";
     }
 
-    private String resolveSuggestedVersion( String baseVersion, String policyId )
+    private String resolveSuggestedVersion( String baseVersion, ReleaseDescriptor releaseDescriptor )
             throws PolicyException, VersionParseException
     {
+        String policyId = releaseDescriptor.getProjectVersionPolicyId();
         VersionPolicy policy = versionPolicies.get( policyId );
         if ( policy == null )
         {
             throw new PolicyException( "Policy '" + policyId + "' is unknown, available: " + versionPolicies.keySet() );
         }
 
         VersionPolicyRequest request = new VersionPolicyRequest().setVersion( baseVersion );
+
+        if ( releaseDescriptor.getProjectVersionPolicyConfig() != null )
+        {
+            request.setConfig( releaseDescriptor.getProjectVersionPolicyConfig().toString() );
+        }
+        request.setWorkingDirectory( releaseDescriptor.getWorkingDirectory() );
+
+        if ( scmRepositoryConfigurator != null && releaseDescriptor.getScmSourceUrl() != null )
+        {
+            try
+            {
+                ScmRepository repository = scmRepositoryConfigurator
+                        .getConfiguredRepository( releaseDescriptor, new Settings() );
```

Comment:
Settings don't matter if they are requied?

Reply:
Good point. Fixing it.

Reply:
Fixed